### PR TITLE
[bugfix/courses-appearance] Global style rebuild and rewrote props

### PR DESCRIPTION
**Pull request by NikitaKhadnevich**
This pull request includes a huge changes on `Courses type` pages and some fix into component:
1. Fixed `Button in Link to Link in Button` logical on Courses and MyCourses page;
2. Adapted Some new style as like `box-shadow` and `justify-content`, `align-self` etc;
3. Rewrote and rebuilt  each style for correctly work on responsive / tablet /mobile resolutions;
4. Rewrote a lot of styles from  `Up` to `Down breakpoints`;
5. Fixed image resolutions and position according Ann's review;
6. Some minor fixed;

### CODE REVIEW

File: frontend/src/pages/MyCourses/MyCoursesList.tsx [hunk 1]
```diff
@@ -83,25 +83,25 @@ const MyCoursesList: React.FC<MyCoursesProps> = ({
...
+                          </Link>
+                        </ContinueTestButton>
                       ) : clientCourse.status === COURSE_STATUSES.pending ? (
                         <StartCourseButton disabled color="primary" variant="mediumContained">
                           Pending
                         </StartCourseButton>
                       ) : (
-                        <Link to={`${PATHS.learnCourse}/${clientCourse._id}`}>
-                          <StartCourseButton color="primary" variant="mediumContained">
+                        <StartCourseButton color="primary" variant="mediumContained">
```
Comment (hauteballe) [inline frontend/src/pages/MyCourses/MyCoursesList.tsx:100]: You can use `<StartCourseButton component={Link} to={${PATHS.learnCourse}/${clientCourse._id}} color="primary" variant="mediumContained">` here

File: frontend/src/components/Layout/styled.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (hauteballe) [inline frontend/src/components/Layout/styled.ts:1]: Remove it please

**Review by hauteballe — CHANGES_REQUESTED** [review]
Unfortunately, the course card doesn't look like as expected (first screenshot is an app, second screen is expected look on 1440 resolution)
image
image

File: frontend/src/components/Course/styled.ts [hunk 3]
```diff
@@ -218,47 +231,74 @@ export const CourseDescription = styled('p')<InfoContainerTypes>(
...
+    height: '50px',
+    display: 'flex',
+    alignItems: 'end !important',
+    alignSelf: 'end !important',
+    paddingBottom: '0px !important',
   },
-  [theme.breakpoints.up('md')]: {
-    height: 'fit-content',
+  [theme.breakpoints.down('lg')]: {
+    height: '50px',
```
Comment (k-krasilnikova) [inline frontend/src/components/Course/styled.ts:246]: Do we really need to duplicate height with the same value?

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/pages/MyCourses/MyCoursesList.tsx (hunk 1 above)
Comment (NikitaKhadnevich) [reply frontend/src/pages/MyCourses/MyCoursesList.tsx]: Retrieve Links from Button and deep its into component

File: frontend/src/components/Layout/styled.ts (hunk 2 above)
Comment (NikitaKhadnevich) [reply frontend/src/components/Layout/styled.ts]: Fixed

**Review by k-krasilnikova — APPROVED** [review]

**Comment by k-krasilnikova** [comment]
Resolve conflicts please!

**Review by irina-sinitsyna — COMMENTED** [review]
@k-krasilnikova At the moment in this branch courses pages at 1024-1440px look like the screenshot, shouldn't they look the same as at 1440+? I think, this view is for tablet only.
image

**Comment by NikitaKhadnevich** [comment]
Fixed each problem moments

**Comment by k-krasilnikova** [comment]
> @k-krasilnikova At the moment in this branch courses pages at 1024-1440px look like the screenshot, shouldn't they look the same as at 1440+? I think, this view is for tablet only. image

Fully agree, they should be as before "2 in row"

**Review by irina-sinitsyna — APPROVED** [review]

File: frontend/src/pages/CoursesList/styled.ts [hunk 4]
```diff
@@ -5,136 +5,142 @@ import { Grid, Box } from '@mui/material';
...
   },
+  display: 'flex',
 });
 
 export const CourseActionsBox = styled(Box)({
   marginLeft: '16px !important',
+  display: 'flex',
+  flexWrap: 'nowrap',
+  [theme.breakpoints.down('xl')]: {
+    marginLeft: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/CoursesList/styled.ts:52]: Should we duplicate it from general styles?

File: frontend/src/pages/CoursesList/styled.ts [hunk 5]
```diff
@@ -5,136 +5,142 @@ import { Grid, Box } from '@mui/material';
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '9px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/CoursesList/styled.ts:70]: Should we duplicate it from general Styles?

File: frontend/src/pages/CoursesList/styled.ts [hunk 6]
```diff
@@ -5,136 +5,142 @@ import { Grid, Box } from '@mui/material';
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '26px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/CoursesList/styled.ts:101]: Should we duplicate it from general Styles?

File: frontend/src/pages/CoursesList/styled.ts [hunk 7]
```diff
@@ -5,136 +5,142 @@ import { Grid, Box } from '@mui/material';
...
   },
 });
+
 export const MobileSearchWrapper = styled('div')({
   width: '100%',
   margin: '16px 0px 8px 0px',
   height: '30px',
+  paddingLeft: '20px !important',
+  [theme.breakpoints.down('lg')]: {
+    paddingLeft: '0px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/CoursesList/styled.ts:142]: Just `paddingLeft: '0 !important'` will be work fine too

File: frontend/src/pages/DetailedCourse/styled.ts [hunk 8]
```diff
@@ -240,52 +243,48 @@ export const StartButton = styled(Button)({
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '9px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/DetailedCourse/styled.ts:253]: Should we duplicate it from general styles?

File: frontend/src/pages/DetailedCourse/styled.ts [hunk 9]
```diff
@@ -240,52 +243,48 @@ export const StartButton = styled(Button)({
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '26px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/DetailedCourse/styled.ts:277]: Should we duplicate it from general styles?

File: frontend/src/pages/MyCourses/styled.ts [hunk 10]
```diff
@@ -5,146 +5,177 @@ import { Link } from 'react-router-dom';
...
   },
+  display: 'flex',
 });
 
 export const CourseActionsBox = styled(Box)({
   marginLeft: '16px !important',
+  display: 'flex',
+  flexWrap: 'nowrap',
+  [theme.breakpoints.down('xl')]: {
+    marginLeft: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/MyCourses/styled.ts:52]: Should we duplicate it from general styles?

File: frontend/src/pages/MyCourses/styled.ts [hunk 11]
```diff
@@ -5,146 +5,177 @@ import { Link } from 'react-router-dom';
...
+    paddingBottom: '5px',
+  },
+  [theme.breakpoints.down('md')]: {
+    marginLeft: '5px !important',
+  },
 });
 
+// Detail Button
```
Comment (irina-sinitsyna) [inline frontend/src/pages/MyCourses/styled.ts:60]: ?

File: frontend/src/pages/MyCourses/styled.ts [hunk 12]
```diff
@@ -5,146 +5,177 @@ import { Link } from 'react-router-dom';
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '9px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/MyCourses/styled.ts:71]: Should we duplicate it from general styles?

File: frontend/src/pages/MyCourses/styled.ts [hunk 13]
```diff
@@ -5,146 +5,177 @@ import { Link } from 'react-router-dom';
...
+  marginRight: '16px !important',
+  fontSize: '16px!important',
+  lineHeight: '22px',
+  padding: '12px!important',
+  [theme.breakpoints.down('xl')]: {
     height: '44px',
-    width: '131px',
-    marginRight: '26px !important',
+    maxWidth: '125px',
+    marginRight: '16px !important',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/MyCourses/styled.ts:102]: Should we duplicate it from general styles?

File: frontend/src/pages/MyCourses/styled.ts [hunk 14]
```diff
@@ -5,146 +5,177 @@ import { Link } from 'react-router-dom';
...
```
Comment (irina-sinitsyna) [inline frontend/src/pages/MyCourses/styled.ts:147]: Should we duplicate it from general styles?

**Review by irina-sinitsyna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**